public/app/features/datasources/state/actions.ts

```diff
@@ -216,6 +237,7 @@ export function addDataSource(
       isDefault: isFirstDataSource,
     };
 
+    // TODO: typo in name
```

Comment:
What actually does this todo mean?

Reply:
I saw this kind of comments in other places. Here the name is `namdExits` instead of `nameExists` but I don't want to fix it in the same PR.